ui/imports/shared/panels/StatusAssetSelector.qml

```diff
@@ -41,6 +45,12 @@ Item {
         id: d
         property string iconSource: ""
         property string text: ""
+        property string tempSearchString
+        property string searchString
+        property Timer inputTextTimer: Timer {
```

Comment:
@Khushboo-dev-cpp if I read this code well... you're using this timer to postpone `assets` model filtering, not to do it while typing but every second? For something like that there is Qt.callLater() function, here: https://doc.qt.io/qt-6/qml-qtqml-qt.html#callLater-method, or you can use the same what we're using in app search check here: https://github.com/status-im/status-desktop/blob/d8abef5b28b7c16507bd7f8ba14030c93f3ef0d0/ui/app/mainui/AppSearch.qml#L12-L14

Reply:
makes sense! will do :)

Reply:
done @saledjenic !